examples/pylab_examples/movie_demo.py

```diff
@@ -22,8 +22,10 @@
     files.append(fname)
 
 print('Making movie animation.mpg - this may take a while')
-os.system("mencoder 'mf://_tmp*.png' -mf type=png:fps=10 -ovc lavc -lavcopts vcodec=wmv2 -oac copy -o animation.mpg")
-#os.system("convert _tmp*.png animation.mng")
+subprocess.call(["mencoder", "'mf://_tmp*.png'", "-mf", "type=png:fps=10",
```

Comment:
I think that with the shell, this is more readable, so I just reverted this part (and deleted the next line)